Source/Core/Common/CDUtils.cpp

```diff
@@ -208,7 +208,7 @@ std::vector<std::string> cdio_get_devices()
 // Returns true if device is a cdrom/dvd drive
 bool cdio_is_cdrom(std::string device)
 {
-#ifndef _WIN32
+#ifdef _WIN32
```

Comment:
Main repo says [it is](https://github.com/dolphin-emu/dolphin/blame/master/Source/Core/Common/CDUtils.cpp#L13)?

Looks like it was [part of a PR merged two days ago](https://github.com/dolphin-emu/dolphin/pull/3102).

How old is the clone of master you're building against?